tasks/prod-rabbitmq-provision.yml

```diff
@@ -0,0 +1,27 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+params:
+  ENV:
+  ADMIN_SERVICE_ACCOUNT_JSON:
+  KUBERNETES_CLUSTER:
+inputs:
+  - name: census-rm-kubernetes-dependencies-repo
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=/root/gcloud-service-key.json
+      export GCP_PROJECT=census-rm-$ENV
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $ADMIN_SERVICE_ACCOUNT_JSON
+      EOL
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+      gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT}
+
+      cd census-rm-kubernetes-dependencies-repo
+
+      RABBITMQ_PRODUCTION_SETUP=true ./setup-rabbitmq.sh
```

Comment:
This seems to be a copy of the other task just with one variable different, can we use the same task and configure that in the pipeline like we do at the moment? I having two near identical tasks makes things easier somehow I don't mind, just seems like what we currently do with a pipeline config var could do this.

Reply:
Made this change. We now pass in if we want to use production setup in the concourse pipelines yaml